examples/util.sh

```diff
@@ -65,6 +65,7 @@ get_lb_ip() {
     ip_var='{.spec.clusterIP}'
   fi
 
-  kubectl --namespace="${namespace}" get service $2 \
-    -o jsonpath=$ip_var
+  kubectl --namespace="${namespace}" port-forward $2 :50000 &
```

Comment:
`kubectl help port-forward` suggests `:50000` chooses a random local port and forwards it to port `50000` on `$2`

Reading these services does not suggest they are listening on port 50000: 

https://github.com/bazelbuild/rules_k8s/blob/002b365cfd200192569fa1ba2b6b3178b693989f/examples/hellohttp/service.yaml#L11-L13

Also I'm not entirely sure but I suspect you may need to change the service type:
https://github.com/bazelbuild/rules_k8s/blob/002b365cfd200192569fa1ba2b6b3178b693989f/examples/hellohttp/service.yaml#L16

Reply:
Looks like hellogrpc listens on a different port:

https://github.com/bazelbuild/rules_k8s/blob/002b365cfd200192569fa1ba2b6b3178b693989f/examples/hellogrpc/service.yaml#L9-L11